app/views/vat/claimVATInterstitialPage.scala.html

```diff
@@ -46,12 +46,11 @@ <h2 class="govuk-heading-s">@messages("claimVATInterstitialPage.p2.title")</h2>
 <p class="govuk-body">@messages("claimVATInterstitialPage.p2.text1")</p>
 
 <h2 class="govuk-heading-s">@messages("claimVATInterstitialPage.p3.title")</h2>
-<p class="govuk-body">@messages("claimVATInterstitialPage.p3.text1")</p>
```

Comment:
Is this page supposed to have a back button?
<img width="1201" height="416" alt="image" src="https://github.com/user-attachments/assets/0dde3f7f-6d28-4ce5-a442-be26b46c1034" />